packages/core/src/modules/oob/OutOfBandService.ts

```diff
@@ -0,0 +1,36 @@
+import type { OutOfBandState } from './domain/OutOfBandState'
+import type { OutOfBandRecord } from './repository'
+
+import { scoped, Lifecycle } from 'tsyringe'
+
+import { OutOfBandRepository } from './repository'
+
+@scoped(Lifecycle.ContainerScoped)
+export class OutOfBandService {
+  private outOfBandRepository: OutOfBandRepository
+
+  public constructor(outOfBandRepository: OutOfBandRepository) {
+    this.outOfBandRepository = outOfBandRepository
+  }
+
+  public async save(outOfBandRecord: OutOfBandRecord) {
+    return this.outOfBandRepository.save(outOfBandRecord)
+  }
+
+  public async updateState(outOfBandRecord: OutOfBandRecord, newState: OutOfBandState) {
+    outOfBandRecord.state = newState
```

Comment:
I see this doesn't include a state yet. Is this something you intend to add in an upcoming PR?

Reply:
I'm sorry, I don't understand what do you mean by this "doesn't include a state yet."?

Reply:
Whoops, I mean event

Reply:
I thought so :) Yes, I plan to add it eventually.